### ISSUE TITLE
Errors in output when no MQTT Broker is available

### ISSUE THREAD

**Issue by gerardwr** [SensorsIot/Wi-Fi-Sniffer-as-a-Human-detector#1]
Great sketch, running fine on Wemos D1 Mini. Thanks for sharing.

I do not have an MQTT broker in my network, so the sketch output contained error messages.

I disable contacting an MQTT broker by commenting out line 98:
```
//    sendDevices();
```

**Comment by frownbreaker** [comment]
When I used this project I had to add 2 dependencies that were missing: Libraries to this project to get it to compile (Wemos D1 Mini)

I had to add these 2 Libraries

1) ArduinoJson https://bblanchon.github.io/ArduinoJson/
2) The Pubsyb MQTT Library https://pubsubclient.knolleary.net

Perhaps add these library dependencies to the read me? Thanks for another great episode!

**Comment by nconder** [comment]
Dependencies please.

**Comment by frownbreaker** [comment]
Hi there I've added the dependencies to my original comment.

"When I used this project I had to add 2 dependencies that were missing:
Libraries to this project to get it to compile (Wemos D1 Mini)

I had to add these 2 Libraries


   1. ArduinoJson https://bblanchon.github.io/ArduinoJson/
   2. The Pubsyb MQTT Library https://pubsubclient.knolleary.net

Perhaps add these library dependencies to the readme? Thanks for another
great episode!

Seth.
